ecosystem/indexer-grpc/indexer-grpc-cache-worker/src/metrics.rs

```diff
@@ -0,0 +1,54 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use aptos_metrics_core::{
+    register_int_counter, register_int_counter_vec, register_int_gauge, IntCounter, IntCounterVec,
+    IntGauge,
+};
+use once_cell::sync::Lazy;
+
+/// Latest processed transaction version.
+pub static LATEST_PROCESSED_VERSION: Lazy<IntGauge> = Lazy::new(|| {
+    register_int_gauge!(
+        "indexer_grpc_cache_worker_latest_processed_version",
+        "Latest processed transaction version",
+    )
+    .unwrap()
+});
+
+/// Number of transactions that saved into cache.
+pub static PROCESSED_VERSIONS: Lazy<IntCounter> = Lazy::new(|| {
+    register_int_counter!(
+        "indexer_grpc_cache_worker_processed_versions",
+        "Number of transactions that have been processed by cache worker",
+    )
+    .unwrap()
+});
+
+/// Number of errors that cache worker has encountered.
+pub static ERROR_COUNT: Lazy<IntCounterVec> = Lazy::new(|| {
+    register_int_counter_vec!(
+        "indexer_grpc_cache_worker_error_count",
+        "Number of errors that cache worker has encountered",
+        &["error_type"]
+    )
+    .unwrap()
+});
+
+/// Data latency for cache worker based on latest processed transaction.
+pub static PROCESSED_LATENCY: Lazy<IntGauge> = Lazy::new(|| {
+    register_int_gauge!(
+        "indexer_grpc_cache_worker_data_latency",
```

Comment:
actually - can you expose this as seconds and suffix the metric name with `_seconds` ? this seems more common in the prometheus ecosystem: https://prometheus.io/docs/practices/naming/

Reply:
also this should be probably a histogram instead of intgauge